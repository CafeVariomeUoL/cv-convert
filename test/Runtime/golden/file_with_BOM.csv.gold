[
{
    "diagnosis": "ORPHA437",
    "condition": "Hypophosphataemic rickets",
    "subject_id": "0"
} , {
    "diagnosis": "ORPHA89936",
    "condition": "X-linked hypophosphataemia",
    "subject_id": "1"
} , {
    "diagnosis": "ORPHA89937",
    "condition": "Autosomal dominant hypophosphataemic rickets",
    "subject_id": "2"
} , {
    "diagnosis": "ORPHA289176",
    "condition": "Autosomal recessive hypophosphataemic rickets",
    "subject_id": "3"
} , {
    "diagnosis": "ORPHA157215",
    "condition": "Hereditary hypophosphataemic rickets with hypercalciuria",
    "subject_id": "4"
} , {
    "diagnosis": "ORPHA352540",
    "condition": "Oncogenic osteomalacia",
    "subject_id": "5"
}
]