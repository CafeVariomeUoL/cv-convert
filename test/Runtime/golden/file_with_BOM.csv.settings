{
  "processFunction": "return {subject_id: `${row.i}`, condition: row.Condition, diagnosis: row.Orphacode}",
  "libraryFunctions": "Lib.calculate_age = function(dob) { \n    var diff_ms = Date.now() - dob.getTime();\n    var age_dt = new Date(diff_ms); \n  \n    return Math.abs(age_dt.getUTCFullYear() - 1970);\n}\n",
  "tsInterface": "export default interface Row {\n    subject_id: string;\n    condition:  string;\n    diagnosis:  string;\n}\n",
  "jsonSchema": {
    "$schema": "http://json-schema.org/draft-07/schema#",
    "definitions": {
      "Row": {
        "$ref": "#/definitions/default"
      },
      "default": {
        "title": "default",
        "type": "object",
        "properties": {
          "subject_id": {
            "type": "string",
            "title": "subject_id"
          },
          "condition": {
            "type": "string",
            "title": "condition"
          },
          "diagnosis": {
            "type": "string",
            "title": "diagnosis"
          }
        },
        "additionalProperties": false,
        "required": [
          "condition",
          "diagnosis",
          "subject_id"
        ]
      }
    },
    "$ref": "#/definitions/default"
  }
}