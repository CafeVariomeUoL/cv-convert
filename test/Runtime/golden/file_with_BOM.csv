Condition,Orphacode
Hypophosphataemic rickets,ORPHA437
X-linked hypophosphataemia,ORPHA89936
Autosomal dominant hypophosphataemic rickets,ORPHA89937
Autosomal recessive hypophosphataemic rickets,ORPHA289176
Hereditary hypophosphataemic rickets with hypercalciuria,ORPHA157215
Oncogenic osteomalacia,ORPHA352540